managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -4012,4 +4018,12 @@ private void cancelScheduledTasks() {
         }
     }
 
+    private void checkInactiveLedgerAndRollOver(final long lastAddEntryTimeMs) {
+        long currentTimeMs = System.currentTimeMillis();
+        if (inactiveLedgerRollOverTimeMs > 0 && currentTimeMs > (lastAddEntryTimeMs + inactiveLedgerRollOverTimeMs)) {
+            log.info("[{}| Closing inactive ledger, last-add entry {}", name, lastAddEntryTimeMs);
+            ledgerClosed(currentLedger);
```

Comment:
I think we should call `rollCurrentLedgerIfFull` method which actually close the ledger and it's the same rollover procedure used for topic when they reach the max quota

Reply:
`rollCurrentLedgerIfFull` closes ledger if its full and that won't be helpful in this case because ledger is not full but ledger inactivity criteria has been triggered.

Reply:
I meant to extract the code block inside the if